routes/reviewRoutes.ts

```diff
@@ -27,4 +39,13 @@ router.post('/', jsonParser, async (req, res) => {
     checkToken(req, res, clientID, next)
 })
 
+router.delete('/', async (req, res) => {
+    const clientID = req.query.clientID,
+        productID = Number(req.query.productID)
+
+    let reviews = await ReviewModel.deleteMany({"userID": clientID, "productID": productID})
```

Comment:
changing await from then should be better so we don't block the execution. and if an user did more reviews to a product, this would delete all reviews, even if the user wants to delete only one review, maybe change it to delete after reviewID